index-scheduler/src/lib.rs

```diff
@@ -1111,6 +1215,16 @@ pub enum TickOutcome {
     WaitForSignal,
 }
 
+/// How many indexes we can afford to have open simultaneously.
+struct IndexBudget {
+    /// Map size of an index.
+    map_size: usize,
+    /// Index count of an index.
+    index_count: usize,
```

Comment:
WHat do you mean by _Index count of an index_?

Reply:
Done https://github.com/meilisearch/meilisearch/pull/3331/commits/92a707b63d3df62d07756ab4d0f7d821c8049d96